docs/requisitos/elicitacao/brainstorming.md

```diff
@@ -78,3 +79,6 @@ Representação do primeiro esboço do fluxo de login e cadastro do tradutor, co
 ### Conclusão
 
 Após a realização do Brainstorming, a equipe concluiu que a formalização dos requisitos seria realizada individualmente, em prol da montagem de backlogs individuais que representassem as abordagens para o desenvolvimento da aplicação esboçada, o que é melhor descrito no documento de [Backlog Geral](../../modelagem/backlog_geral).
+
+### Referências
```

Comment:
Put a reference from where you were guided to describe what is a brainstorming process. Also, this link does not describe the material used to be based on.